mozilla_django_oidc/middleware.py

```diff
@@ -0,0 +1,82 @@
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
+
+import time
+
+import django
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.utils.crypto import get_random_string
+
+from mozilla_django_oidc.utils import absolutify, import_from_settings
+
+
+# Django 1.11 makes changes to how middleware work. In Django 1.11+, we want to
+# use the mixin so that our middleware works as is.
+if tuple(django.VERSION[0:2]) >= (1, 10):
+    from django.utils.deprecation import MiddlewareMixin
+else:
+    class MiddlewareMixin(object):
+        pass
+
+
+# FIXME(willkg): This doesn't appear to be used anywhere.
+def logout_url():
+    """Log out the user from Auth0."""
+    url = 'https://' + import_from_settings('OIDC_OP_DOMAIN') + '/v2/logout'
+    url += '?' + urlencode({
+        'returnTo': import_from_settings('LOGOUT_REDIRECT_URL', '/'),
+        'client_id': import_from_settings('OIDC_RP_CLIENT_ID')
+    })
+    return url
```

Comment:
This doesn't appear to be used anywhere so I added a FIXME to it. If it turns out it is used, then I can remove it.